src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -78,7 +80,7 @@ public static void SetCurrentCircuitHost(CircuitHost circuitHost)
             Renderer.UnhandledSynchronizationException += SynchronizationContext_UnhandledException;
         }
 
-        public string CircuitId { get; } = Guid.NewGuid().ToString();
+        public string CircuitId { get; }
```

Comment:
Is the circuitId considered a secret? IE can we log it? @blowdart 

We will **need** to have identifiers that we can use for correlation and troubleshooting.

Reply:
I don't necessarily think of them as a secret, but I also don't feel super confortable logging them.
I say we can change this to a struct and have two fields, one with the actual payload and one with a string version of the random bytes for logging purposes.

The random bytes are ok to use as the identifier for display/logging purposes as they are data protected and someone would need access to the dp key to actually generate the over the wire format.

Reply:
No, it's not, and, because its sent over a data protected payload even if you knew it, you'd also have to know the signing keys in order to abuse it and hop circuits.